src/archivematicaCommon/lib/archivesspace/client.py

```diff
@@ -486,27 +487,46 @@ def add_digital_object(self, parent_archival_object, dashboard_uuid, title="", i
             "notes": [{
                 "jsonmodel_type": "note_digital_object",
                 "type": "originalsloc",
-                "content": [dashboard_uuid]
+                "content": [dashboard_uuid],
+                "publish": True,
             }],
             "restrictions": restricted,
             "subjects": parent_record['subjects'],
             "linked_agents": parent_record['linked_agents'],
         }
 
+        i = 0
+        for pnote in parent_record["notes"]:
+            content = []
+            for subnote in pnote["subnotes"]:
+                content.append(subnote["content"])
+
+            new_object["notes"].append({
+                "jsonmodel_type": "note_digital_object",
+                "type": pnote["type"],
```

Comment:
In d8f2090 , I have added a hardcoded array, copied from the ArchivesSpace code, rather than connect to the ArchivesSpace api and check.  I did this in order for testing, I'll need to go back now and change this to an api call.